iati/resources.py

```diff
@@ -57,88 +61,89 @@
 """The name of a file containing an Organisation Schema."""
 
 
-def get_codelist_paths(version=iati.version.STANDARD_VERSION_ANY):
+@iati.version.decimalise_integer
+@iati.version.allow_possible_version
+def get_codelist_paths(version):
     """Find the paths for all Codelists at the specified version of the Standard.
 
     Args:
-        version (str): The version of the Standard to return the Codelists for. Defaults to iati.version.STANDARD_VERSION_ANY.
+        version (str / int / Decimal / iati.Version): The version of the Standard to return the Codelists for.
+            Decimal: Return paths for the specified version of the Standard.
+            Integer: Return paths for the latest Decimal version within the given integer.
+            Version-independent: Return paths for Non-Embedded Codelists.
```

Comment:
wrt to version independent info, is this expected?

```
>>> iati.resources.get_codelist_paths(iati.version.STANDARD_VERSION_ANY)
[]
```

Reply:
That is... not the documented behaviour!

There are a couple of issues around the naming of the folder containing NE Codelists (being `codelists_non_embedded/` rather than `version_independent/codelists/`) and the `default` module currently expecting an empty array.

I'd suggest that management and handling of NE Codelists should be looked at as standalone Issue (with this particular docstring being updated to reflect the current behaviour).

Reply:
...the problem with `default` is due to `STANDARD_VERSION_ANY` being added as a thing after it was tested, so there not being any tests against this value (so it's deem to not be a permitted value at all!)

Reply:
https://github.com/IATI/pyIATI/pull/297/commits/b26c2c61e475b6c0187e4f390064a204516836eb